Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deltachat"
-version = "1.0.0-alpha.1"
```

Comment:
why are you changing the version?

Reply:
Because I needed a newer version to be able to get the flatpak build working, which needs to build from a tag.  So I bumped the version and tagged it.

Reply:
lets please stop doing this, in PRs, cc @hpk42 this makes for really annoying merge conflicts :(